modules/accessing-hosts-on-aws.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * networking/accessing-hosts.adoc
+
+[id="accessing-hosts-on-aws_{context}"]
+= Accessing hosts on Amazon Web Services (AWS)
+
+The {product-title} installer does not create any public IP addresses for any of
+the Amazon Elastic Compute Cloud (Amazon EC2) instances that it provisions for
+your {product-title} cluster. In order to be able to SSH to your {product-title}
+hosts, you must follow this procedure.
+
+.Procedure
+
+. Create a security group that allows SSH access into the virtual private cloud
+(VPC) created by the `openshift-install` command.
+
+. Create an Amazon EC2 instance inside the VPC that the installer created or on
+one of the public subnets the installer created.
+
+. Associate a public IP address with the Amazon EC2 instance that you created.
++
+Unlike with the {product-title} installation, you must associate the Amazon EC2
+instance you created with an SSH keypair. It does not matter what operating
```

Comment:
Depends on which AMI you use.  With RHCOS, you can provide keys via Ignition, like the installer does.  So we should at least soften the "must" or more clearly scope it to a particular AMI family.